src/test/java/com/sonyericsson/hudson/plugins/gerrit/trigger/mock/MockGerritHudsonTriggerConfig.java

```diff
@@ -375,7 +420,7 @@ public int getBuildScheduleDelay() {
 
     @Override
     public int getDynamicConfigRefreshInterval() {
-        return 30;
+        return Config.DEFAULT_DYNAMIC_CONFIG_REFRESH_INTERVAL;
```

Comment:
Is this really a good idea? Or is that one `30` as well? :)

Reply:
The following commit adds a new variable and uses it hence. yes the value is 30. It was mostly triggered to fix a check style warning. It might be possible to fix it using annotations. 

https://github.com/jenkinsci/gerrit-trigger-plugin/pull/413/commits/90a2eb84559876ffcaffc5319df7af42a52495e0